SlidingSegmentedControlTests/SlidingSegmentedControlTests.swift

```diff
@@ -82,6 +82,16 @@ class SlidingSegmentedControlTests: XCTestCase {
         XCTAssertEqual(slidingSegmentedControl.imageViews.map {$0.contentMode}, [.center, .center, .center])
     }
 
+    func testImageViewsIsAccessibilityElement() {
+        let slidingSegmentedControl = SlidingSegmentedControl(images: [UIImage(), UIImage()])
+        XCTAssertEqual(slidingSegmentedControl.imageViews.map { $0.isAccessibilityElement }, [true, true])
```

Comment:
I don't think this simple `map` should be considered complex or _clever_ code. I use it because it's concise.

Consider this example:
```Swift
    func testImageViewsImagesV1() {
        let images = [UIImage(), UIImage()]
        let slidingSegmentedControl = SlidingSegmentedControl(images: images)
        XCTAssertEqual(slidingSegmentedControl.imageViews.flatMap { $0.image }, images)
    }

    func testImageViewsImagesV2() {
        let images = [UIImage(), UIImage()]
        let slidingSegmentedControl = SlidingSegmentedControl(images: images)
        guard slidingSegmentedControl.imageViews.count == 2 else {
            XCTFail("Expected 2 image views")
            return
        }
        XCTAssertEqual(slidingSegmentedControl.imageViews[0].image, images[0])
        XCTAssertEqual(slidingSegmentedControl.imageViews[1].image, images[1])
    }
```
If I don't do guard, test will be trap with EXC_BAD_INSTRUCTION if `imageViews` haven't been populated, so for a good test, I do need it. Hence this, I try to test against arrays themselves when possible and doing a simple `map` or a `flatMap` feels a lot better than making test twice as big.

In the particular example that you commented on the difference in size is minimal, I agree, but we're all about consistency right (we either allow maps in tests or we don't)?

By the way - using _clever_ code is not considered good also for production (because it is harder to maintain). So if we consider any map to be _clever_ code, we would simply need to avoid it everywhere, but I doubt you'd want that. ;)